src/main/java/nb/kafka/operator/KafkaUtilities.java

```diff
@@ -65,6 +65,10 @@
 
   private AdminClient adminClient;
 
+  final int maxTries = 10;
```

Comment:
I suggest to rename this `private final static int MAX_TRIES`  to follow naming convention of ALL_CAPS for final statics (see [here](https://github.com/nbogojevic/kafka-operator/blob/master/src/main/java/nb/kafka/operator/ConfigMapManager.java) for example)